darshan-util/darshan-parser.c

```diff
@@ -52,17 +52,29 @@
 /*
  * Datatypes
  */
+
+/* Structure to accumulate per-file derived metrics, regardless of how many
+ * ranks access it. The "file_hash_table" UT hash table data structure is
+ * used to keep track of all files that have been encountered in the log.
+ * The *_accum_file() functions are used to iteratively accumulate metrics,
+ * while the *_file_list() functions are used to emit them to stdout.
+ */
 typedef struct hash_entry_s
 {
     UT_hash_handle hlink;
     darshan_record_id rec_id;
     int64_t type;
     int64_t procs;
     void *rec_dat;
-    double cumul_time;
-    double slowest_time;
+    double cumul_io_total_time; /* cumulative metadata and io time */
+    double slowest_io_total_time; /* slowest rank metadata and io time */
```

Comment:
i think "cumulative metadata and rw time" and "slowest rank metadata and rw time" for the comments here

Reply:
fixed in 5cbf2e2d..602c286e